### ISSUE TITLE
crash on startup - libzopfli.dll missing

### ISSUE THREAD

**Issue by JMiha** [Lymphatus/caesium-image-compressor#25]
Hi, just flagging a simple bug with (presumably) a simple fix.

The latest windows x64 release crashes on startup, saying libzopfli.dll is missing. My little hack to make it work was to copy zopfli.dll and rename it to libzopfli.dll .

Thanks for the app, very useful :)

**Comment by Lymphatus** [comment]
I think it's the same as #18 but seems odd because I'm pretty sure I updated the `alpha.2` win package with the correct .dll. Which version gave you the issue?

**Comment by raidsan** [comment]
caesium-image-compressor-2.0.0-alpha.2-win.zip

**Comment by XiaofengdiZhu** [comment]
the same

**Comment by hahaleyile** [comment]
Well, you can rename the file in the folder where exe file is. Then the problem solved.

**Comment by rudolphos** [comment]
Same problem.
Windows 10 64b, 2004 version

![image](https://user-images.githubusercontent.com/4366810/83781138-c34b0400-a696-11ea-9f07-6e39fd875ed4.png)

```
---------------------------
Caesium Image Compressor.exe - System Error
---------------------------
The code execution cannot proceed because libzopfli.dll was not found. Reinstalling the program may fix this problem. 

```

**Comment by 3xploiton3** [comment]
same problem

![image](https://user-images.githubusercontent.com/19517680/86269446-f203be00-bbf3-11ea-9b1d-eb0c70d2e56b.png)

**Comment by 3xploiton3** [comment]
> Same problem.
> Windows 10 64b, 2004 version
> 
> ```
> ---------------------------
> Caesium Image Compressor.exe - System Error
> ---------------------------
> The code execution cannot proceed because libzopfli.dll was not found. Reinstalling the program may fix this problem. 
> ```
> 
> Renaming fixed the problem.

it worked
![image](https://user-images.githubusercontent.com/19517680/86269654-41e28500-bbf4-11ea-8450-034d4d0d3640.png)

**Comment by 1024mb** [comment]
problem is still happening, the dll is still missing from the latest zip package. apprently the file has been wrongly named, the file `zopfli.dll` should be named `libzopfli.dll`.

**Comment by Lymphatus** [comment]
Will be fixed in the next release

**Comment by theAkito** [comment]
Can confirm.

**Comment by mrkaban** [comment]
the same problem in version 2.0.0-alpha.2-vin, I attach screenshots
![111](https://user-images.githubusercontent.com/14289360/90137447-d4537800-dd9f-11ea-9ac8-95533926b041.png)
![222](https://user-images.githubusercontent.com/14289360/90137449-d584a500-dd9f-11ea-9209-b99ea2a4fcd1.png)

**Comment by Lymphatus** [comment]
Should be resolved with `v2.0.0-alpha.3`.
